trace_stats/main.cpp

```diff
@@ -120,13 +121,15 @@ void GatherAndPrintStats(const Dive::CaptureMetadata &meta_data, std::ostream &o
     // Number of CpWaitForIdle()
     // Number of prefetches
     // Number of loads/stores to GMEM
-    enum Stats
+    enum Stats : uint32_t
```

Comment:
consider enum classes since they're scoped (so don't pollute the current scope).
```suggestion
    enum class Stats : uint32_t
```

Reply:
this is a bit painful to add `class` in this case. with this change, every where the enum is used as uint32_t needs to be explicitly cased with static_cast<> which makes the code harder to read (there are quite a lot in this file).

Reply:
actually i found a better solution by using a struct instead, this way i could add the scope without forcing the static_cast